src/data/scripts/hullmods/vayra_modular_shields.java

```diff
@@ -384,7 +375,7 @@ private void doShieldEmitterStuff(ShipAPI ship) {
                         && ship.getParentStation() != null
                         && ship.getParentStation().equals(check.getParentStation())) {
                     generator = check;
-                    storedGenerators.put(ship, generator);
+                    ship.setCustomData(STORED_GENERATORS_KEY, generator);//localData.storedGenerators.put(ship, generator);
```

Comment:
same. just so we're on the same page - we don't want inline comments to tell us *nothing* about the *WHY*.

comments are there to tell us about the *WHY* not about *HOW* or especially not about *HOW WE USED TO*.
just get rid of all of this commented out code.